### PR TITLE
[EDA] Student and Faculty Record Type Settings

### PR DESCRIPTION
**Pull request by skosgi1**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: force-app/main/default/lwc/courseConnectionSettings/courseConnectionSettings.js [hunk 1]
```diff
@@ -74,7 +86,6 @@ export default class CourseConnectionSettings extends LightningElement {
     courseConnectionSettingsWire(result) {
         this.courseConnectionSettingsWireResult = result;
         if (result.data) {
-            console.log(result.data);
```
Comment (xinnanlisf) [inline force-app/main/default/lwc/courseConnectionSettings/courseConnectionSettings.js:77]: good catch!

File: force-app/main/default/classes/CourseConnectionSettingsVMapper_TEST.cls [hunk 2]
```diff
@@ -126,5 +150,110 @@ private with sharing class CourseConnectionSettingsVMapper_TEST {
...
+                new RecordTypeModel('000000000000000003', 'Inactive Record Type 2', 'Inactive_Record_Type_2', false)
+            );
+
+            return recordTypeModelsById;
+        }
+    }
+    /**************************************************************************************************************************************
+     * @description Stub class to simulate the response from RecordTypeMapper to return an empty map of Account Record Types
+     **************************************************************************************************************************************/
+    private class STUB_RecordTypeMapperEmpty extends RecordTypeMapper {
```
Comment (xinnanlisf) [inline force-app/main/default/classes/CourseConnectionSettingsVMapper_TEST.cls:254]: I dont think this stub is being used.

Comment (xinnanlisf) [reply force-app/main/default/classes/CourseConnectionSettingsVMapper_TEST.cls:254]: And i do think we need to cover these edge cases.

**Review by xinnanlisf — CHANGES_REQUESTED** [review]
Left one comment. Otherwise it looks good.

File: force-app/main/default/classes/CourseConnectionSettingsVMapper_TEST.cls (hunk 2 above)
Comment (skosgi1) [reply force-app/main/default/classes/CourseConnectionSettingsVMapper_TEST.cls]: You're right, thanks for catching. Removed it.

**Review by xinnanlisf — APPROVED** [review]
Approved!

**Comment by spinkelman** [comment]
QA Passes - Merging

**Pull request merged**